src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6591,4 +6591,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_UnreadRecordParameter_Title" xml:space="preserve">
     <value>Parameter is unread. Did you forget to use it to initialize the property with that name?</value>
   </data>
+  <data name="ERR_RecordAmbigCtor" xml:space="preserve">
+    <value>A record cannot be declared with a single positional parameter with the type of that record.</value>
```

Comment:
>with the type of that record [](start = 74, length = 28)

"of the same type as the record"?

Reply:
I think it would be better to report more specific error. Some thing like: "A copy constructor cannot be synthesized for a record with the only positional parameter of the same type as the record." 

---
In reply to: [543383571](https://github.com/dotnet/roslyn/pull/49975#discussion_r543383571) [](ancestors = 543383571)

Reply:
"Synthesized copy constructor conflicts with a primary constructor." is even better, I think. 

---
In reply to: [543408078](https://github.com/dotnet/roslyn/pull/49975#discussion_r543408078) [](ancestors = 543408078,543383571)